# AvinelandiiDJ
Modeling of carbon and nitrogen partitioning in the heterotrophic diazotroph Azotobacter vinelandii DJ 

Diego Tec Campos1,2, Cristal Zuñiga1, Anurag Passi1, John Del Toro3, Juan D. Tibocha-Bonilla4, Alejandro Zepeda2, 
Michael J. Betenbaugh3, Karsten Zengler1,5,6*

1Department of Pediatrics, University of California, San Diego, 9500 Gilman Drive, La Jolla, CA 92093-0760, USA
2Facultad de Ingeniería Química, Universidad Autónoma de Yucatán, Mérida, Yucatán, México	
3Department of Chemical and Biomolecular Engineering, The Johns Hopkins University, 3400 North Charles Street, Baltimore, MD 21218, USA 
4Bioinformatics and Systems Biology Graduate Program, University of California, San Diego, La Jolla CA 92093-0412, USA.
5Department of Bioengineering, University of California, San Diego, La Jolla CA 92093-0412, USA.
6Center for Microbiome Innovation, University of California, San Diego, 9500 Gilman Drive, La Jolla, CA 92093-0403, USA

Abstract

Nitrogen fixation is an important metabolic process carried out by microorganisms, which converts molecular nitrogen into 
inorganic nitrogenous compounds such as ammonia (NH3). These nitrogenous compounds are crucial for biogeochemical cycles and 
for the synthesis of essential biomolecules, i.e. nucleic acids, amino acids and proteins. Azotobacter vinelandii is a 
bacterial non-photosynthetic model organism to study aerobic nitrogen fixation (diazotrophy) and hydrogen production. 
Moreover, the diazotroph can produce biopolymers like alginate and polyhydroxybutyrate (PHB) that have important industrial 
applications. However, many metabolic processes such as partitioning of carbon and nitrogen metabolism in A. vinelandii remain 
unknown to date.
Genome-scale metabolic models (M-models) represent reliable tools to unravel and optimize metabolic functions at genome-scale. 
M-models are mathematical representations that contain information about genes, reactions, metabolites and their associations. 
M-models can simulate optimal reaction fluxes under a wide variety of conditions using experimentally determined constraints. 
Here we report on the development of a M-model of the wild type bacteria A. vinelandii DJ (iDT1278) which consists of 
2,003 metabolites, 2,469 reactions, and 1,276 genes. We validated the model using high-throughput phenotypic and physiological 
data, testing 180 carbon sources and 95 nitrogen sources. iDT1278 was able to achieve an accuracy of 89% and 91% for growth 
with carbon sources and nitrogen source, respectively. This comprehensive M-model will help to comprehend metabolic processes 
associated with nitrogen fixation, ammonium assimilation, and production of organic nitrogen in an environmentally important 
microorganism. 
